pandas/io/formats/format.py

```diff
@@ -498,9 +502,15 @@ def __init__(
         else:
             self.columns = frame.columns
 
+        self.truncate_h = None  # type: Optional[int]
```

Comment:
If this gets assigned to an int subsequently and `None` isn't valid I think should just type as `int`.

There's some syntax trickery at play here due to py 3.5 support as in py36 forward you could declare this without initializing I think like:

```
self.truncate_h: int
self.truncate_v: int
```

But can't do that in 3.5

Reply:
`None` is valid, so that we get the same behaviour as `False` when it was `bool` 

`None` is when the DataFrame is not truncated. the assigned `int` is the truncated row/column number.